src/app/stacks/visionstack.cpp

```diff
@@ -34,10 +39,26 @@ VarList * VisionStack::getSettings() {
 }
 
 void VisionStack::process(FrameData * data) {
-  for (auto p : stack) {
-    p->lock();
-    p->process(data,opts);
-    p->unlock();
+  if(_v_print_timings->getBool()) {
```

Comment:
This looks like it will print the stats for every frame, for every camera. That is a lot of stdout processing. I recommend the numbers be accumulated and averaged over (for example) 30 frames, and printed only every 0.5 seconds.

Reply:
my workflow was like this: starting ssl-vision, selecting the desired camera and activate the "print timings" flag for a short time. Then I had a look at the console to check the timings.

Having this flag active for a longer time will probably never be useful.

Accumulating results could be nice, but it would also increase complexity in the code and might hide spikes in the processing times. So I'd rather leave it like it is now.

Reply:
That sounds fine, as long as we disclose this: that it is not intended to be used during actual games, or running continuously with the timings, and that it is most useful only running in single-camera mode for a test period.

Reply:
ok. Do you mean to document this somewhere? I'm not sure, where I should do this. The flag is false by default, so it must be switched on actively per camera.
If this is fine for you, I would merge the PR?